src/pytest_rabbitmq/factories/executor.py

```diff
@@ -37,7 +37,8 @@ def __init__(
         envvars = {
             "RABBITMQ_LOG_BASE": logpath + f"/rabbit-server.{port}.log",
             "RABBITMQ_MNESIA_BASE": path + "mnesia",
-            "RABBITMQ_ENABLED_PLUGINS_FILE": plugin_path + "/plugins",
+            "RABBITMQ_ENABLED_PLUGINS_FILE": plugin_path + "/enabled",
+            "RABBITMQ_PLUGINS_DIR": plugin_path,
```

Comment:
🤔  maybe skip both settings if the plugin's dir does not exists? question, what happens if we'll point to the real files and paths, will it work?

Reply:
my thinking is that there's not much point to point to the default for both of these settings (enabled_plugin_file and plugins_dir) if they are not existing.